.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -41,13 +41,14 @@ wget https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning/master
 python collect_env_details.py
 ```
 
- - PyTorch Version (e.g., 1.0):
- - OS (e.g., Linux):
- - How you installed PyTorch (`conda`, `pip`, source):
- - Build command you used (if compiling from source):
+ - PyTorch Lightning Version (e.g., 1.3.0):
+ - PyTorch Version (e.g., 1.0)
```

Comment:
nit: we should pick a PyTorch version that Lightning supports for the template default 
```suggestion
 - PyTorch Lightning Version (e.g., 1.3.0):
 - PyTorch Version (e.g., 1.8)
```